server/service-limiter.js

```diff
@@ -0,0 +1,48 @@
+const debug = require('debug');
+
+const logger = require('./logger');
+
+const nonEssentialRobots = ['Ahrefs', 'Bluechip Backlinks', 'Semrush', 'SpiderFoot', 'WebMeUp'];
+
+const essentialRobots = ['Facebook', 'Pingdom', 'Twitter'];
+
+const debugServiceLevel = debug('serviceLevel');
+
+let serviceLevel = 0;
```

Comment:
:icecream: Suggesting for consistency:
```suggestion
const debugLib = require('debug');
const logger = require('./logger');

const debug = debugLib('serviceLevel');
const NON_ESSENTIAL_ROBOTS = ['Ahrefs', 'Bluechip Backlinks', 'Semrush', 'SpiderFoot', 'WebMeUp'];
const ESSENTIAL_ROBOTS = ['Facebook', 'Pingdom', 'Twitter'];

let serviceLevel = 0;
```